packages/edge-login-ui-rn/src/components/themed/FourDigitInput.js

```diff
@@ -0,0 +1,143 @@
+// @flow
```

Comment:
The word "Four" shouldn't be a part of any file/folder/component name.
We might change it tomorrow to 5, or 6, or whatever. It should be kept general and the length of the Pin Code should be a prop.